lib/runners/processor/clang_tidy.rb

```diff
@@ -31,21 +31,27 @@ def analyzer_bin
 
     def analyze(changes)
       issues = []
+      analyzed_files = []
 
       changes
         .changed_paths
         .select { |path| path.fnmatch?(GLOB_SOURCES, File::FNM_EXTGLOB | File::FNM_CASEFOLD) }
-        .map{ |path| relative_path(working_dir / path, from: current_dir) }
+        .map { |path| relative_path(working_dir / path, from: current_dir) }
         .reject { |path| path.to_s.start_with?("../") } # reject files outside the current_dir
         .each do |path|
           stdout, = capture3!(analyzer_bin, path.to_s, "--", *option_include_path,
             is_success: ->(status) { [0, 1].include?(status.exitstatus) })
           construct_result(stdout) { |issue| issues << issue }
+          analyzed_files << path
         end
 
-      Results::Success.new(guid: guid, analyzer: analyzer).tap do |result|
-        result.add_issue(*issues)
+      if analyzed_files.empty?
+        trace_writer.message "No files to analyze."
+      else
+        trace_writer.message "#{analyzed_files.size} file(s) were analyzed."
       end
```

Comment:
[suggestion] a ternary operator gives you  just one line of code

```suggestion
      trace_writer.message analyzed_files.empty? ? "No files to analyze." : "#{analyzed_files.size} file(s) were analyzed."
```

Reply:
Make sense. 👍